src/hooks/index.test.js

```diff
@@ -295,7 +295,7 @@ describe('#useMemoCompare', () => {
 
 describe('#useFeatureFlag', () => {
   let wrapper, store;
-  const knownProperty = 'I18N_ENABLED';
+  const knownProperty = 'EFB_FORM_SCHEMA_SUPPORT_ENABLED';
```

Comment:
This is just a string used for unit testing the `useFeatureFlag` hook. We were using the `I18N_ENABLED` flag for testing purposes. Now that it was removed, I changed it for the `EFB_FORM_SCHEMA_SUPPORT_ENABLED` flag. Tests pass ok.